src/integration-test/resources/InvoiceSenderIT/__files/test3_processInvoices/mappings/api-citizen-get-person.json

```diff
@@ -0,0 +1,11 @@
+{
+	"request": {
+		"method": "GET",
+		"urlPattern": "/citizen/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}"
+	},
+	"response": {
+		"headers": {"Content-Type": "application/json"},
+		"status": 200
```

Comment:
No response body?

Reply:
No, we are using this just to check for status code. If a person have protected identity, the response code will not be 200. 

```
@GetMapping(path = "/{personId}", produces = ALL_VALUE)
ResponseEntity<Void> getPerson(@PathVariable("personId") String personId);
```